content/docs/guides/crosswalk/kubernetes/_index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Pulumi Crosswalk for Kubernetes"
+linktitle: Crosswalk for Kubernetes
+menu:
+  userguides:
+    parent: guides
+    identifier: crosswalk-kubernetes
+
+aliases: ["/docs/reference/crosswalk/kubernetes/"]
```

Comment:
This alias doesn't make sense. Did we used to have content at this URL path?

Shouldn't it be:

```suggestion
aliases: ["/docs/guides/k8s-the-prod-way/"]
```

?

Reply:
The aliases were an artifact of copy/paste that I stubbed out w/o thinking through. I've removed all of the aliases on the new crosswalk content.